File: apendice/2.py
"""
def pad(x):
	al=False
	count=0
	for i in str(x):
		if i==".":
			al=True
		if al:
			if
cantidad = int(input())
for x in range(cantidad):
    score, chart = map(float, input().split())
    if score >= 10000000:
        modifier = 2
    elif score <= 9800000:
        modifier = (score - 9500000) / 300000
    else:
        modifier = 1 + (score - 9800000) / 200000
    print(round(modifier + chart,7))
    #print('{0:.7f}'.format(modifier + chart))
"""
cantidad = int(input())
for x in range(cantidad):
    score, chart = map(float, input().split())
    if score >= 10000000:
        modifier = 2
    elif score <= 9800000:
        modifier = (score - 9500000) / 300000
    else:
        modifier = 1 + (score - 9800000) / 200000
    ans=modifier + chart
    print('%.7f' %ans)#,end="\n")
    #print('{0:.7f}'.format(modifier + chart))

File: apendice/2.cpp

//print('{0:.7f}'.format(modifier + chart))

#include <bits/stdc++.h>
using namespace std;
int main(){
	double score,chart,modifier;
	int cantidad;cin>>cantidad;
	for(int i=0;i<cantidad;i++){
	  cin>>score;
	  cin>>chart;
	  if (score>=10000000){
	  	modifier = 2;
	  }
	  else if(score <= 9800000){
	  	modifier = (score - 9500000) / 300000;
	  }
	  else{
	 		modifier = 1 + (score - 9800000) / 200000;
	  }
	 cout<<(modifier + chart)<<endl;
	}
}